.github/workflows/e2e-windows.yml

```diff
@@ -159,6 +162,8 @@ jobs:
             project: e2e-angular-core
           - node_version: 16
             project: e2e-angular-extensions
+          - node_version: 16
+            project: e2e-angular-module-federation
```

Comment:
It should also be added for `node_version: 20`

Reply:
it is? right? 
<img width="642" alt="image" src="https://github.com/nrwl/nx/assets/12140467/a6710f09-c219-4cb2-a0e7-8811e478a351">